compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/uikit/Events.uikit.kt

```diff
@@ -16,8 +16,26 @@
 
 package androidx.compose.ui.uikit
 
+import androidx.compose.ui.input.key.KeyEvent
+import androidx.compose.ui.input.key.internal
 import androidx.compose.ui.input.pointer.PointerEvent
 import platform.UIKit.UIEvent
+import platform.UIKit.UIPress
+
+@Deprecated("Renamed to uiEventOrNull", ReplaceWith("uiEventOrNull"))
+val PointerEvent.uikitEventOrNull: UIEvent? get() = uiEventOrNull
```

Comment:
The name of this function should represent the package/framework, not the name of the class. By the same reason we have `awtEventOrNull()`, not `eventOrNull()`.

So, I wouldn't rename `uikitEventOrNull`, it looks good to me.